microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -855,5 +953,22 @@ todo: how select element, where?  that instruction is prominent in UI, so clarif
 -->
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 116.
-*  This checkbox is present in Microsoft Edge Stable 114.
+*  This checkbox is present in Microsoft Edge Canary 117.
+*  This checkbox is present in Microsoft Edge Stable 116.
+
+
+<!-- ====================================================================== -->
+## Enable Storage Buckets Tree in Application panel
+
+_new as of Microsoft Edge 118_
+
+Whether to enable the Storage Buckets Tree in the **Application** tool.
+<!-- todo: need text -->
```

Comment:
@captainbrosset @zoherghadyali - need real text for Experiments checkbox "Enable Storage Buckets Tree in Application panel"

looking for See Also links